### PR TITLE
update values to snake case

### PR DESCRIPTION
**Pull request by jenniemc**
## Description
Change the feature toggle values in `featureFlagNames.js` to be in snake case to match the values in `feature.yml`

## Original issue(s)
department-of-veterans-affairs/va.gov-team#29917

## Testing done
n/a

## Screenshots
n/a

## Acceptance criteria
- [ ] change all vaos feature toggle values to be in snake_case

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by timwright12 — APPROVED** [review]

File: src/platform/utilities/feature-toggles/featureFlagNames.js [hunk 1]
```diff
@@ -87,25 +87,25 @@ export default Object.freeze({
   useLighthouseFormsSearchLogic: 'new_va_forms_search',
   vaGlobalDowntimeNotification: 'va_global_downtime_notification',
   vaOnlineFacilitySelectionV22: 'vaOnlineFacilitySelectionV22',
```
Comment (timwright12) [inline src/platform/utilities/feature-toggles/featureFlagNames.js:89]: @jenniemc is this one VAOS as well?

Comment (jenniemc) [reply src/platform/utilities/feature-toggles/featureFlagNames.js:89]: @timwright12 Looks like we do reference `vaOnlineFacilitySelectionV22` in our app. I'll make the change and push the update to this PR.

**Review by jbalboni — APPROVED** [review]
This looks good to me for the moment. I'm pretty sure will be other VAOS updates to make during the process of getting rid of the duplicate flags, so we'll want to keep an eye on that.

File: src/platform/utilities/feature-toggles/featureFlagNames.js (hunk 1 above)
Comment (timwright12) [reply src/platform/utilities/feature-toggles/featureFlagNames.js]: thanks!

**Review by timwright12 — APPROVED** [review]

**Pull request merged**